subprojects/distributions-integ-tests/src/integTest/groovy/org/gradle/AllDistributionIntegrationSpec.groovy

```diff
@@ -36,7 +36,7 @@ class AllDistributionIntegrationSpec extends DistributionIntegrationSpec {
 
     @Override
     int getMaxDistributionSizeBytes() {
-        return 220 * 1024 * 1024
+        return 222 * 1024 * 1024
```

Comment:
:harold:

Reply:
Yep.  I also don't think these dist size tests are serving a real purpose other than build friction - I've mentioned this before elsewhere.

I think instead, we should have tests that compare the current size with the size of version n-1 and fail if it has grown more than X% without manual approval.

Reply:
Note that I improved the output so for the future, you will know how much you need to raise each number